Mathlib/GroupTheory/GroupAction/SubMulAction/OfStabilizer.lean

```diff
@@ -84,22 +84,30 @@ theorem neq_of_mem_ofStabilizer (a : α) {x : ofStabilizer G a} : ↑x ≠ a :=
   x.prop
 
 @[to_additive]
-lemma Enat_card_ofStabilizer_eq_add_one (a : α) :
+lemma ENat_card_ofStabilizer_add_one_eq (a : α) :
     ENat.card (ofStabilizer G a) + 1 = ENat.card α := by
   dsimp only [ENat.card]
   rw [← Cardinal.mk_sum_compl {a}, map_add, add_comm, eq_comm]
   congr
   simp
 
+@[deprecated  (since := "2025-07-15")]
+alias Enat_card_ofStabilizer_eq_add_one := ENat_card_ofStabilizer_add_one_eq
+
 @[to_additive]
-lemma nat_card_ofStabilizer_eq [Finite α] (a : α) :
```

Comment:
I think that it would be kind to deprecate this result, explaining in the deprecation comment that the statement is a little different, but can still be used!